modules/account-lib/src/coin/stx/abstractContractBuilder.ts

```diff
@@ -0,0 +1,85 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import BigNum from 'bn.js';
+import {
+  ClarityValue,
+  ContractCallOptions,
+  makeUnsignedContractCall,
+  PayloadType,
+  PostCondition,
+  PostConditionMode,
+  UnsignedContractCallOptions,
+  UnsignedMultiSigContractCallOptions,
+} from '@stacks/transactions';
+import { TransactionType } from '../baseCoin';
+import { BuildTransactionError, InvalidParameterValueError, InvalidTransactionError } from '../baseCoin/errors';
+import { Transaction } from './transaction';
+import { TransactionBuilder } from './transactionBuilder';
+
+export abstract class AbstractContractBuilder extends TransactionBuilder {
+  private _options: UnsignedContractCallOptions | UnsignedMultiSigContractCallOptions;
+  protected _contractAddress: string;
+  protected _contractName: string;
+  protected _functionName: string;
+  protected _postConditionMode?: PostConditionMode;
+  protected _postConditions?: PostCondition[];
+  protected _functionArgs: ClarityValue[];
+  private _anchorMode: number;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this._anchorMode = 3;
```

Comment:
Why here the value is 3? Can we add a doc specifying this value?

Reply:
maybe i can use this https://github.com/stacksgov/sips/blob/main/sips/sip-005/sip-005-blocks-and-transactions.md#transaction-encoding